siliconcompiler/apps/sc_issue.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2023 Silicon Compiler Authors. All Rights Reserved.
+import sys
+import os
+import siliconcompiler
+import tarfile
+import json
+import importlib
+
+def main():
+    progname = "sc-issue"
+    chip = siliconcompiler.Chip(progname)
+    switchlist = ['-cfg',
+                  '-version',
+                  '-arg_step',
+                  '-arg_index',
+                  '-tool_task_variable',
+                  '-tool_task_option',
+                  '-loglevel']
+    description = """
+    -----------------------------------------------------------
+    Restricted SC app that generates a sharable testcase from a 
+    failed flow or runs an issue generated with this program.
+
+    To generate a testcase, use:
+        sc-issue -generate -cfg <cfg_file>
```

Comment:
This is a minor thing, but when I first ran this I accidentally pointed at the final manifest in the job directory (since it was a successful run) and did not include -arg_step and -arg_index, which made it fail out with a slightly gnarly error.

I think making a note to point that you need to point to a step-specific manifest (maybe just embedding a path fragment in the example like below) might be helpful, and/or including error checking for step/index being defined.

```
sc-issue -generate <stepdir>/outputs/<design>.pkg.json
```

Reply:
good point